releasenotes/notes/0.6/expval_pauli-ec2ce5650dd82c49.yaml

```diff
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    Adds Pauli expectation value snapshot support to the `"density_matrix"`
+    simulation method of the :class:`qiskit.providers.aer.QasmSimulator`.
+    Add snapshots to circuits using the
+    :class:`qiskit.providers.aer.extensions.SnapshotExpectationValue`
+    extension.
+upgrade:
```

Comment:
I'm not sure if this is an upgrade note. The upgrade section is normally for something a user may need to know when upgrading from a previous versions because it requires manual intervention. Things like removals, default behavior changes, etc. I would probably put this under `features` or one of the other categories

```suggestion
```